aces/pipeline_scripts/override_tclean_commands.json

```diff
@@ -2499,12 +2499,36 @@
   },
   "Sgr_A_st_i_03_TM1": {
     "tclean_cube_pars": {
+      "spw25": {
+        "nchan": 1912,
+        "width": "244119.45Hz",
+        "threshold": "0.01Jy"
+      }
```

Comment:
you need trailing commas here.  Do a syntax check on this file before submitting - you can use the command-line tool `jq`, i.e., do `jq . override_tclean_commands.json`.  You can also try loading the file into python with 
```python
import json
with open('override_tclean_commands.json', 'r') as fh:
    data = json.load(fh)
```

Reply:
Thanks, Adam! I will submit a new pull request.